scripts/insert_simple_simulation.py

```diff
@@ -0,0 +1,46 @@
+from src.base_model import db
+from src.implementor.models import SimulationConfiguration
+from src.schedule.schedule_configuration import (
+    ScheduleConfiguration,
+    ScheduleConfigurationXSimulationPlatform,
+)
+from src.spawner.spawner import (
+    SpawnerConfiguration,
+    SpawnerConfigurationXSchedule,
+    SpawnerConfigurationXSimulationConfiguration,
+)
+
+with db.atomic():
+    platforms = ["bs_0", "bs_6"]
+    regular_schedule = ScheduleConfiguration.create(
+        schedule_type="TrainSchedule",
+        strategy_type="RegularScheduleStrategy",
+        strategy_start_tick=1000,
```

Comment:
I think it should start sooner, so you can see what happens quickly.
```suggestion
        strategy_start_tick=100,
```